eng/Subsets.props

```diff
@@ -310,6 +310,11 @@
     <ProjectToBuild Include="$(CoreClrProjectRoot)nativeaot\BuildIntegration\BuildIntegration.proj" Category="clr" />
   </ItemGroup>
 
+  <!-- Running the tests for browser wasm requires BuildIntegration -->
+  <ItemGroup Condition="$(_subset.Contains('+clr.nativeaotlibs+')) and ('$(TargetOS)' == 'Browser' and '$(TargetArchitecture)' == 'wasm')">
+    <ProjectToBuild Include="$(CoreClrProjectRoot)nativeaot\BuildIntegration\BuildIntegration.proj" Category="clr" />
+  </ItemGroup>
+
```

Comment:
I guess I added that because we dont ask for `clr.nativeaotruntime` at line 52 (hyperlink not working):
```
 Run `build clr.nativeaotlibs+libs -c [Debug|Release] -a wasm -os Browser`
```

Reply:
Let me try without it.

Reply:
reverted

Reply:
I think #2146 added it already.